contrib/kml_export.lua

```diff
@@ -348,7 +365,7 @@ local function create_kml_file(storage, image_table, extra_data)
         else
             kmlFileOpenCommand = "xdg-open "..exportDirectory.."/\""..exportKMLFilename.."\""
 	end
-        dt.control.execute( kmlFileOpenCommand) 
+        coroutine.yield("RUN_COMMAND", kmlFileOpenCommand) 
```

Comment:
@elman22 Yes, we know. The version from @boucman only work with the latest git version. We are trying to find a solution that works with both versions without the need to change all scripts